### ISSUE TITLE
Option to load contract state with a block number of tx hash

### ISSUE THREAD

**Issue by axic** [ethereum/remix-ide#1215]
Since during the `at address` "operation" (is there a name for this?) storage values are loaded off the blockchain, it would make sense to have an option specifying the required starting state.

This should be possible since `debug_storageAt` accepts a block number, IIRC?

**Comment by ninabreznik** [comment]
Would it help if you search the transaction in the terminal? There under
details you can find state. But not sure if it's rhere after you clear
terminal window... Is this what you had in mind?

**Comment by axic** [comment]
No, this about loading the contract state off the blockchain, which is done when using the `at address` button.

**Comment by ninabreznik** [comment]
Ah, ok. Btw. we currently came to the conclusion to call 'at address' - access. And 'create' is deploy.

So we have deploy and access.

We're working on improving UI again in this tab, so you can take a look and give some feedbacks if you have time. https://github.com/ethereum/remix-ide/issues/1153
